compiler/next/test/resolution/testScopeResolve.cpp

```diff
@@ -476,6 +693,11 @@ int main() {
   test6();
   test7();
   test8();
+  test9();
+  test10();
+  test11();
+  test12();
+  test13();
```

Comment:
It also might be nice to ensure use statements that don't enable qualified access still allow unqualified access (e.g. `use M as _;`) unless that's intentionally not handled yet.

Reply:
I think the test could check that but it doesn't work yet. I'd like to leave it for future work. So I'll note it in the PR message.